index.ts

```diff
@@ -114,7 +114,7 @@ export enum Key {
     Multiply = '*',
     Add = '+',
     Subtract = '-',
-    DecimalPoint = '.',
+    DecimalPoint = 'Decimal',
```

Comment:
This would break in Chrome/FF/Safari ... It would be better to introduce a new key maybe something like `MSDecimalPoint = 'Decimal'`